creator-node/src/routes/files.js

```diff
@@ -317,6 +319,13 @@ const getDirCID = async (req, res) => {
 }
 
 module.exports = function (app) {
+  app.get('/track_content', handleResponse(async (req, res) => {
```

Comment:
i think this name collision might be confusing can we do /track_content_status or something?

Reply:
hmm I think it's appropriate cus at least it ties back relevancy to /track_content route. I don't feel strongly here

Reply:
i think it's going to confuse someone (it confused me at first) and will need an explainer that polling is when the method is GET.

Reply:
ok, will update